suitcase/mongo_normalized/__init__.py

```diff
@@ -53,15 +52,14 @@ def _create_indexes(self):
 
         If the index already exists, this has no effect.
         """
-        self._resource_collection.create_index('uid')
+        self._resource_collection.create_index('uid', unique=True)
         self._resource_collection.create_index('resource_id')  # legacy
         # TODO: Migrate all Resources to have a RunStart UID, and then make a
         # unique index on:
         # [('uid', pymongo.ASCENDING), ('run_start', pymongo.ASCENDING)]
         self._datum_collection.create_index('datum_id', unique=True)
         self._datum_collection.create_index('resource')
-        self._run_start_collection.create_index(
-            [('uid', pymongo.DESCENDING)], unique=True)
```

Comment:
@tacaswell Do you have any idea why we made this a `DESCENDING` index? It's the same way [in the original in `databroker.v0`](https://github.com/bluesky/databroker/blob/c2dfc4e478a4cf90a979f5b0f74f9e7ee29f4898/databroker/headersource/mongo.py#L52-L53). I'm going to assume it was overly-aggressive pattern-matching or carelessness unless I'm forgetting something.

Reply:
I agree, the order should only matter for indexs that involve time or scan_id.  My guess is copy-paste lazyness.

Reply:
Thanks for the careful read, @gwbischof. You have cleaned up a 5-year-old mistake.